src/Controls/src/Core/TypedBinding.cs

```diff
@@ -85,7 +85,11 @@ public TypedBinding(Func<TSource, (TProperty value, bool success)> getter, Actio
 
 			_handlers = new PropertyChangedProxy[handlers.Length];
 			for (var i = 0; i < handlers.Length; i++)
+			{
+				if (handlers[i] is null)
+					continue;
 				_handlers[i] = new PropertyChangedProxy(handlers[i].Item1, handlers[i].Item2, this);
```

Comment:
This loop accesses `handlers[i]` a couple times. Is it worth storing in a local instead of indexing into it multiple times?

Reply:
it's an array, not a list, and we don't foreach for perf reasons. we could cache it, but I'm quite confident this won't save a second over a lifetime of running :)